pkg/controllers/disruption/suite_test.go

```diff
@@ -1361,6 +1366,358 @@ var _ = Describe("Candidate Filtering", func() {
 	})
 })
 
+var _ = Describe("Metrics", func() {
+	var nodePool *v1beta1.NodePool
+	var labels = map[string]string{
+		"app": "test",
+	}
+	BeforeEach(func() {
+		nodePool = test.NodePool(v1beta1.NodePool{
+			Spec: v1beta1.NodePoolSpec{
+				Disruption: v1beta1.Disruption{
+					ConsolidationPolicy: v1beta1.ConsolidationPolicyWhenUnderutilized,
+					// Disrupt away!
+					Budgets: []v1beta1.Budget{{
+						Nodes: "100%",
+					}},
```

Comment:
nit: this seems to be a common Nodepool configuration in our tests. Thoughts on adding another test package helper for greating highDisruptingNodepool? this way we don't keep rewriting this every test?

Reply:
For now, I think this is fine. This is really us just wanting to override our default that is set for production (10%). It's not clear to me if setting a value other than 10% in our default NodePool is a good idea or not, so it seems fine to me to just override it specifically when we need it